src/components/interface/ConfirmationModal.vue

```diff
@@ -14,17 +14,17 @@
         <PasswordInput show-eye />
```

Comment:
Setting the input's placeholder to empty might be good to remove the repetitive prompt 🤔 

<img width="457" alt="Screen Shot 2021-09-20 at 14 01 53" src="https://user-images.githubusercontent.com/22848332/134075404-6e3f2b51-8f5e-4ae6-8294-e81091074183.png">

Reply:
Changed modal body text from "Enter password" to "Confirm your password" (the input placeholder text is part of a different component).